events/LAR_Spain.txt

```diff
@@ -853,6 +869,12 @@ country_event = {
 							infantry = { x = 1 y = 0 }
 							infantry = { x = 1 y = 1 }
 							infantry = { x = 1 y = 2 }
+							infantry = { x = 2 y = 0 }
```

Comment:
why is this like a 44 width or something lol

Reply:
should be a 40 width if i didnt make a mistake